matcher/core.py

```diff
@@ -10,6 +12,14 @@
     'FairFlow' : FairFlow
 }
 
+class MATCHER_STATUS(Enum):
```

Comment:
can we call it `MatcherStatus`, I think that is the convention for class names.

Reply:
Agreed, done.